Document checkbox value (#39215)
mdn/content: files/en-us/web/api/htmlinputelement/value/index.md

## Value

> A string containing the value of the {{htmlelement("input")}} element, or the empty string if the input element has no value set, unless the input element has a type of `checkbox` or `radio`, in which case it will be the string `"on"` if it has no value set (although the value can still be manually set to the empty string).

Reviewer: ```suggestion
A string specifying the default value of the {{htmlelement("input")}} element.
```

Author: I'm a little confused, to be honest? This seems like the opposite of where I was trying to do. And this also seems even less correct to me. This is the entry for the `.value` class property of the JavaScript `HTMLInputElement` DOM interface, not the HTML `value` attribute.
